docs/topics/strings.md

```diff
@@ -111,14 +111,18 @@ By default, a pipe symbol `|` is used as margin prefix, but you can choose anoth
 ## String templates
 
 String literals may contain _template expressions_ – pieces of code that are evaluated and whose results are concatenated into the string.
-A template expression starts with a dollar sign (`$`) and consists of either a name:
+They can be used with variables of any type - `toString()` is invoked implicitly to produce the result.
+A template expression starts with a dollar sign (`$`) and consists of either a variable name:
 
 ```kotlin
 fun main() {
 //sampleStart
     val i = 10
     println("i = $i") 
     // i = 10
+    val letters = listOf("a","b","c","d","e")
+    println("List of letters: $letters") // evaluates as letters.toString() 
+    // List of letters: [a, b, c, d, e]
```

Comment:
It's better to mention something with an explicit `toString` defined to make the example more obvious. Like this:

```
val obj = object { override fun toString(): String = "Hello world" }
println("obj = $obj")
// obj = Hello world
```

Reply:
Oh, really, that's very nice - I'm not so familiar with anonymous objects so I couldn't even think of such an example. Maybe we should keep my example and add yours also?

Reply:
> Maybe we should keep my example and add yours also?

What benefit does `listOf` example add?

Reply:
> What benefit does `listOf` example add?

I supposed that demonstration of all three behaviours (let's call it three levels of difficulty) will be most illustrative:
* Easy level - A simple int 
* Medium level -  A collection type 
* Hard level - An overridden toString() - thank you for this contribution

I won't insist on adding listOf() example if I don't convince you with this comment